index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <header>
+    <nav id="navbar">
+      <a href="#welcome-section"> Home </a>
+      <a href="#projects"> Projects </a>
```

Comment:
- [ ] A good practice is to group `<a>` tags within `<ul>` tags, each hyperlink being a `<li>` element